exercise_04.py

```diff
@@ -0,0 +1,24 @@
+def found_pi(number):
+    formula = 0
+
+    for i in range(1, number + 1):
+        formula += (-1) ** ( i + 1) / (2 * i - 1)
+        pi = 4 * formula
+
+    return f"{pi:.4f}"
+
+
+def print_pi_values_table(n):
```

Comment:
@DanielVelinrb , again lying to the user. You are not printing a table, you are printing the header, and then you are returning a string. Could you please fix it at the point you just **print** the table?